lib/kubeclient/common.rb

```diff
@@ -202,6 +205,14 @@ def resource_name(entity_type)
         entity_type.pluralize.downcase
       end
 
+      def proxy_url_service(namespace, service, protocol)
```

Comment:
@jimmidyson tbh, I just looked at it as a string as it's part of the path.
Do you think it's better to get both parameters separably and optionally?